examples/tina-cloud-starter/components/tina-wrapper.tsx

```diff
@@ -76,4 +73,82 @@ const Inner = (props) => {
   )
 }
 
+import { Cloudinary } from 'cloudinary-core'
+import {
+  Media,
+  MediaList,
+  MediaListOptions,
+  MediaStore,
+  MediaUploadOptions,
+} from 'tinacms'
+
+export class CloudinaryMediaStore2 implements MediaStore {
+  accept = '*'
+  private api: Cloudinary
+
+  constructor(public cloud_name: string) {
+    this.api = new Cloudinary({
+      cloud_name: this.cloud_name,
```

Comment:
Do we need to set `this.cloud_name = cloud_name` first?

Reply:
Oh.... that's not supposed to be there. Some old debugging code.

Reply:
Yeah sorry I realize that this code was elshewhere after I commented